.circleci/config.yml

```diff
@@ -37,9 +37,6 @@ commands:
       - restore_cache: # restores saved cache if no changes are detected since last run
           keys:
             - cimg-go-pkg-mod-<<parameters.os>>-{{ checksum "go.sum" }}
-      - persist_to_workspace:
-          root: ~/
-          paths: go/pkg/mod
```

Comment:
On linux where we have multiple tests attaching to the same workspace this is already called in setup job.

Reply:
And on windows we don't need it because no other job attaches to the same workspace.